seqr/utils/vcf_utils.py

```diff
@@ -72,7 +83,7 @@ def validate_vcf_and_get_samples(vcf_filename):
     _validate_vcf_header(header)
     if not samples:
         raise ErrorsWarningsException(['No samples found in the provided VCF.'], [])
-    _validate_vcf_meta(meta)
+    _validate_vcf_meta(meta, genome_version)
```

Comment:
is `meta` ever an empty dict by this point? we could check to see if it's empty and skip calling this function when it is to avoid executing the first 10 lines of `_validate_vcf_meta()`

Reply:
no this will always be populated, the way VCFs work is if there is a FORMAT field there are also FORMAT meta fields, and we already validate above that the field exists overall